test/print-pretty-error.test.ts

```diff
@@ -42,4 +42,8 @@ describe('#printPrettyError()', () => {
 
 		expect(consoleResult).toContain('Error message content');
 	});
+
+	test('Returns the message text', () => {
+		expect(printPrettyError('Error message content')).toContain('Error message content');
```

Comment:
This is a stricter check:

```suggestion
		expect(printPrettyError('Error message content')).toEqual('Error message content');
```